src/api/api.ts

```diff
@@ -132,7 +132,7 @@ export const callGetTxByPuzzleHash = (params: {
     status?: number
 }) =>
     apiHandler({
-        url: '/addon/get_tx_by_puzzle_hash',
+        url: '/transactions',
```

Comment:
It seems that the endpoint of `callGetTxByPuzzleHash` is wrong.
<img width="1348" alt="CleanShot 2023-04-26 at 21 31 09@2x" src="https://user-images.githubusercontent.com/55678466/234591405-9e2831e7-e023-422b-b8ea-0d2007834a94.png">